RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,20 @@
+# Release checklist
+
+## Documentation
+- [] Write simplified release notes for FEC with instructions for review
+- [] Update documentation in any relevant repo
+
+### Communciations
+- [] Are there communications around this release from 18F or FEC?
+
+## Reviews
+- [] 18F QA review on staging
+- [] FEC QA review on staging
+- [] File accessibility review request [here](https://github.com/18F/Accessibility_Reviews/issues/new)
+
+## Deployement
+- [] Rebuild tables if necessary
```

Comment:
Maybe change this to `Run update scripts in standalone app if necessary` to better reflect what we had to do.

In addition, add the following:

[] Deploy API
[] Deploy Web App
[] Deploy CMS